src/hooks/useArticle.js

```diff
@@ -3,7 +3,14 @@ import { useI18n } from 'hooks'
 import { getArticle, getArticleMediaList, getSuggestedArticles } from 'api'
 import { canonicalizeTitle } from 'utils'
 
+let currentArticle

```

Comment:
Adding global state like that is not great. It took me a while to figure out the whole logic here. It's clever, and it works, but I would like us to explore more conventional (or "boring") options.

One such option is to give ArticlePreview the required info about the source on open
```
showArticlePreview({ title, lang, dir, <info about the current article> })
```

Reply:
I start with the conventional options, but then it's really bad =( but anyway, let me try again with it.

Reply:
indeed, yes, `useArticleLinksNavigation` has `galleryItems` that also come from the source, we can have everything as the `source`, so it can pass to `showArticlePreview`